src/yachtzee/diet.ts

```diff
@@ -267,6 +267,13 @@ export function executeNextDietStep(stopBeforeJellies?: boolean): void {
         const entry = dietUtil.dietArray.find((entry) => entry.name === name);
         if (entry) {
           if (entry.fullness > 0) {
+            if (
+              have($skill`Aug. 16th: Roller Coaster Day!`) &&
+              !get("_aug16Cast") &&
+              myFullness() > 0
+            ) {
+              useSkill($skill`Aug. 16th: Roller Coaster Day!`);
+            }
```

Comment:
I don't know if we want to blindly cast this one

Reply:
Same with the other one. Unless it happens to be that day, we don't know that we definitely want to cast it.

Reply:
I'm working off the assumption that this is probably the last thing we're going to be doing - we would have grabbed items in dailies and buffs prior to yachtzeeing, so if we deemed those less profitable than a stomach space/Lucky! charge, then we shouldn't have any issues using those here

Reply:
I suppose we might want to save 1 charge of the scepter skills for offhand remarkable if we aren't ascending

Reply:
If you add that I'll merge this in right away. I have an image for what I want support to look like now, so we're all good